plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraTable.java

```diff
@@ -0,0 +1,193 @@
+    {
+        ImmutableList.Builder<String> partitionColumns = ImmutableList.builder();
+        ImmutableList.Builder<String> clusterColumns = ImmutableList.builder();
+        for (ColumnDefinition columnDefinition : columnDefinitions) {
+            if (columnDefinition.primaryKeyType == PrimaryKeyType.PARTITION) {
+                partitionColumns.add(columnDefinition.name);
+            }
+            else if (columnDefinition.primaryKeyType == PrimaryKeyType.CLUSTER) {
+                clusterColumns.add(columnDefinition.name);
+            }
+        }
+        List<String> partitions = partitionColumns.build();
+        List<String> clusters = clusterColumns.build();
+
+        checkState(partitions.size() > 0, "Cassandra table must have at least one partition key");
+
+        String primaryKey = format(
+                "(%s%s)",
+                partitions.stream().collect(joining(", ", "(", ")")),
+                clusters.stream().collect(joining(", ", clusters.isEmpty() ? "" : ", ", "")));
+
+        return columnDefinitions.stream()
+                .map(columnDefinition -> columnDefinition.name + " " + columnDefinition.type + (columnDefinition.static0 ? " static" : ""))
+                .collect(joining(",\n", "(\n", ",\nPRIMARY KEY " + primaryKey + "\n)\n" + tableOptions.orElse("")));
+    }
+
+    private static List<String> columnNames(List<ColumnDefinition> columnDefinitions)
+    {
+        return columnDefinitions.stream()
+                .map(columnDefinition -> columnDefinition.name)
+                .collect(toImmutableList());
+    }
+
+    public static ColumnDefinition partitionColumn(String name, String type)
+    {
+        return new ColumnDefinition(name, type, false, PrimaryKeyType.PARTITION);
+    }
+
+    public static ColumnDefinition clusterColumn(String name, String type, boolean static0)
+    {
+        return new ColumnDefinition(name, type, static0, PrimaryKeyType.CLUSTER);
+    }
+
+    public static ColumnDefinition generalColumn(String name, String type)
+    {
+        return new ColumnDefinition(name, type, false, PrimaryKeyType.GENERAL);
+    }
+
+    public static List<String> columnsValue(int rowsCount, List<Function<Integer, String>> columnsValueProviders)
+    {
+        ImmutableList.Builder<String> rows = ImmutableList.builder();
+        for (int rowNumber = 1; rowNumber <= rowsCount; rowNumber++) {
+            ImmutableList.Builder<String> rowValues = ImmutableList.builder();
+            for (Function<Integer, String> columnValues : columnsValueProviders) {
+                rowValues.add(columnValues.apply(rowNumber));
+            }
+            rows.add(join(",", rowValues.build()));
+        }
+        return rows.build();
+    }
+
+    public static class ColumnDefinition
+    {
+        private final String name;
+        private final String type;
+        private final boolean static0;
```

Comment:
What is static0? Can it be renamed to something that clarifies what it is?

Reply:
> What is static0?

Cassandra supports static columns, which you can refer to [column-definition](https://cassandra.apache.org/doc/latest/cassandra/cql/ddl.html#column-definition)

> Can it be renamed to something that clarifies what it is?

sharedValueInPartition? Do you have any suggestions?

Reply:
Ah, then let's just rename to `static`. Thanks for the link - it seems like a storage optimisation done by Cassandra.

Reply:
> let's just rename to static

`static` is a keyword in Java.

Reply:
> static is a keyword in Java.

stupid me 🤦

`is_static`? `static_column`?

Reply:
I prefer the latter, but it requires camel naming.

Reply:
`staticColumn`.